test/setup_test_psqlgraph.py

```diff
@@ -114,6 +114,6 @@ def create_indexes(host, user, password, database):
     )
 
     args = parser.parse_args()
-    setup_database(args.user, args.password, args.database)
+    setup_database(args.user, args.password, args.database, args.host)
```

Comment:
Use the `host` arg for setup, which is helpful for local testing against a Docker postgres image.